microsoft-365/compliance/create-a-custom-sensitive-information-type.md

```diff
@@ -107,12 +107,28 @@ Use this procedure to create a new sensitive information type that is based on a
 7. You can choose to edit or remove the existing patterns and add new ones. Choose the default confidence level for the new pattern. The values are **Low confidence**, **Medium confidence**, and **High confidence**.
 8. Choose and define **Primary element**. The primary element can be a **Regular expression**, a **Keyword list**, a **Keyword dictionary**, or one of the pre-configured **Functions**. See, [What the DLP functions look for](what-the-dlp-functions-look-for.md).
 9. Fill in a value for **Character proximity**.
-10. (Optional) If you have **Supporting elements** or any **Additional checks** add them. If needed you can group your **Supporting elements**.
+10. (Optional) If you have **Supporting elements** or any **Additional checks** add them. If needed you can group your **Supporting elements** in groups of "All of these", "Any of these" with ability to select how many and "Not any of these".
 11. Choose **Create**.
 12. Choose **Next**.
 13. Choose the **recommended confidence level** for this sensitive information type.
 14. Check your setting and choose **Submit**.
 
+## Additional Checks
+**Exclude specific matches** allow you to define keywords that will be excluded when detecting matches for this pattern of the info type. For example, you might exclude test credit card numbers like '4111111111111111' so that they're not matched as a valid number.
```

Comment:
```suggestion
**Exclude specific matches** allow you to define keywords that will be excluded when detecting matches for this pattern of the sensitive information type. For example, you might exclude test credit card numbers like '4111111111111111' so that they're not matched as a valid number.
```